src/utility/VersionUtility.ts

```diff
@@ -61,10 +61,8 @@ export class VersionUtility {
         //  2. Is a flight in progress? Get the binaries version from ExP.
         //  3. Fall back onto whichever LKS version was specified in package.json. (For dev and staging, this is whichever version matches this extension. For Prod, this is a variable we specify in the Mindaro-Connect-VSCode pipeline.)
         const packageJsonVersion: string = packageJsonContent[`extensionMetadata`][`expectedCLIVersion`];
-        const expectedCLIVersion = process.env.BRIDGE_CLI_VERSION != null ? process.env.BRIDGE_CLI_VERSION :
-            (expBinariesVersion != null && bridgeEnvironment === Environment.Production ? expBinariesVersion :
-            (process.env.BRIDGE_BUILD_PATH != null ? null :
```

Comment:
💡  just a fyi: https://github.com/Azure/vscode-bridge-to-kubernetes/pull/10#discussion_r987439099 thanks.

Reply:
@daniv-msft we need your suggestions here, @Vidya2606 from our team is making these changes. we have refactored the condition to following removing the condition for `process.env.BRIDGE_BUILD_PATH != null ? null`because expectedCLIVersion has to be null if BRIDGE_BUILD_PATH is set right? 
`const expectedCLIVersion = (bridgeEnvironment === Environment.Production) ? packageJsonVersion : 
              (process.env.BRIDGE_CLI_VERSION != null) ? process.env.BRIDGE_CLI_VERSION : null;`

Reply:
Minor comment, but we should likely allow developers to set and use BRIDGE_CLI_VERSION even if they're in Production environment. For example, it will be useful if we need to repro a bug with the current version that was not reproducing with the previous one. Here, if you're in Production, you will be stuck with whatever is the packageJsonVersion (or you'll need to update the VS Code's extension code, but that's not fun either).

Regarding dropping BRIDGE_BUILD_PATH, it's probably fine. As mentioned on the other PR, this is needed if someone wants to update both the VS Code extension and the CLI at the same time. Now that these products are separated in two repos, that's likely a less important scenario.

Reply:
My other feedback however is that if we're dropping BRIDGE_BUILD_PATH, it would be great to update the other files/readmes that reference it.